src/vm/domainfile.cpp

```diff
@@ -3203,6 +3203,9 @@ void GetTimeStampsForNativeImage(CORCOMPILE_VERSION_INFO * pNativeVersionInfo)
 #endif // FEATURE_CORECLR
 }
 
+static constexpr uint32_t CmovFlags = 0x00008001;
+static constexpr uint32_t Sse2Flags = 0x04000000;
```

Comment:
Nit: `Cmov` and `Sse2` are always spelled all capital `CMOV` and `SSE2` everywhere else in CoreCLR